config/config.exs

```diff
@@ -20,6 +20,10 @@ config :mobile_app_backend, MBTAV3API.ResponseCache,
   gc_interval: :timer.hours(1),
   allocated_memory: 250_000_000
 
+config :mobile_app_backend, MBTAV3API.RepositoryCache,
+  gc_interval: :timer.hours(2),
+  allocated_memory: 250_000_000
```

Comment:
Setting to 2 hours based on the recommendation [here](https://hexdocs.pm/nebulex/Nebulex.Adapters.Local.html#module-caveats-when-using-ttl-option) that ttl should be less than gc_interval